app/scripts/directives/unit.coffee

```diff
@@ -24,10 +24,16 @@ angular.module('swarmApp').directive 'unit', ($log, game, commands, options, uti
       #util.utcdoy 1000 * secs
       if isFinite estimate.val
         nonlinear = if not (estimate.unit?.isVelocityConstant?() ? true) then 'less than ' else ''
-        secs = moment.duration(estimate.val, 'seconds').humanize()
-        if /\ years$/.test secs
-          secs = 'a long time'
-          nonlinear = ''
+        secs = moment.duration(estimate.val, 'seconds')
+        if options.showAccurateTiming()
+           secs = secs.format("Y [yr] M [mth] d [day] h [hr] m [min] s [sec]")
+           nonlinear = ''
```

Comment:
Time estimates are linear - `cost / production`. (The code calls production "velocity" in some places.) This means estimates are exact for energy/larvae/territory, but usually too low for everything in the meat tab, where units-producing-units means production's always increasing. So, until estimates for meat units are improved (#304 - wow, I never created an issue for this?), we simply punt and put "less than" before any estimate where production's increasing over time.

Estimates didn't change in this commit, so we still need to signal in some way that this time isn't exact. (I don't want bug reports saying "estimates are wrong".) At the very least, "less than" needs to stay - it's a bit more awkward when you're showing exact times though, one reason I hadn't touched #277 yet!